### PR TITLE
Feature/footer

### PR DESCRIPTION
**Pull request by kylelengling**
Basic footer implementation & updates of footer component folder structure to match our new implementation standard

### CODE REVIEW

File: components/footer/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import Image from 'next/image';
+import styles from '../../styles/Home.module.css';
+import { classnames } from 'tailwindcss-classnames';
+
+const Footer = (): JSX.Element => {
+  return (
+    <footer className={classnames('bg-charcoal', 'flex', 'text-white', 'justify-center', 'py-12')}>
```
Comment (bhgeorge) [inline components/footer/footer.tsx:7]: We should probably make a container component so we can make sure the X margins on this match across the site.

File: components/footer/footer.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
+import Image from 'next/image';
+import styles from '../../styles/Home.module.css';
```
Comment (bhgeorge) [inline components/footer/footer.tsx:2]: This can get ripped out.

File: components/footer/footer.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        <div className={classnames('flex', 'flex-col', 'mb-4')}>
+          <a href="/help">
+            <h2>Need help?</h2>
+          </a>
+          <a href="https://developers.sitecore.com" target="_blank" rel="noopener noreferrer">
+            Powered by{' '}
+            <span className={classnames('ml-2')}>
+              <Image src="/sitecore-white.svg" alt="Sitecore Logo" width={72} height={16} />
+            </span>
+          </a>
```
Comment (bhgeorge) [inline components/footer/footer.tsx:18]: Next Image is kind of a butt. Was just gonna say add some classes to the anchor to make the sitecore image vertically centered... but again Next Image doesn't place nice with that.

```
<a className="flex items-center" href="https://developers.sitecore.com" target="_blank" rel="noopener noreferrer">
  <span className="mr-2">Powered by{' '}</span>
  <Image src="/sitecore-white.svg" alt="Sitecore Logo" width={72} height={16} />
</a>
```

Here's the absolutely crap workaround I found

**Review by bhgeorge — CHANGES_REQUESTED** [review]
Are we missing these links from the image or are those omitted from this?

I'm guessing omitted because only the first column matters?

Screen Shot 2021-09-13 at 7 19 13 AM

File: components/footer/footer.tsx (hunk 1 above)
Comment (bhgeorge) [reply components/footer/footer.tsx:7]: Well, I should have looked at the Trello board first. I am making this now. I'll add it to the footer if this PR goes in first.

File: components/footer/footer.tsx (hunk 3 above)
Comment (kylelengling) [reply components/footer/footer.tsx:18]: Ty Brian -- will add when I get a chance after investigating the global css issue.

File: components/footer/footer.tsx (hunk 2 above)
Comment (kylelengling) [reply components/footer/footer.tsx:2]: Will do

File: components/footer/footer.tsx (hunk 1 above)
Comment (kylelengling) [reply components/footer/footer.tsx:7]: Ty for the container update -- and re--content/links - yeah, they're omitted. I asked Ders about what we wanted for content and I understood it as the Need Help image/link and then socials/copyright info from the live site.

File: components/footer/footer.tsx (hunk 2 above)
Comment (kylelengling) [reply components/footer/footer.tsx:2]: Do we still need to rip Image out if we're using it for the implementation of the footer image?

Comment (bhgeorge) [reply components/footer/footer.tsx]: Ah I meant `styles` not Image. I should have been more clear.

**Review by bhgeorge — APPROVED** [review]

File: components/footer/footer.tsx (hunk 2 above)
Comment (kylelengling) [reply components/footer/footer.tsx]: npnp -- removed styles

**Comment by andershopkins** [comment]
@kylelengling 

- The component should be renamed to `Footer.tsx` and not `footer.tsx`
- The `Footer` component should be in `/components/site/Footer`. 
- The "Need Help?" link should have some separation from the content below. Say...`mb-8` and be underlined, e.g.  ` className={classnames('mb-8', 'underline')}`

Once that's done, the rest is good.

**Review by andershopkins — CHANGES_REQUESTED** [review]
@kylelengling

- The component should be renamed to `Footer.tsx`
- The Footer component should be in `/components/site/Footer`.
- The "Need Help?" link should have some separation from the content below. Say...`mb-8` and be underlined, e.g. `className={classnames('mb-8', 'underline')}`

Once that's done, the rest is good.

**Review by andershopkins — APPROVED** [review]

**Pull request merged**